src/plugin.js

```diff
@@ -163,22 +148,12 @@ function zoomNumericalScale(scale, zoom, center, zoomOptions) {
 	var minDelta = newDiff * minPercent;
 	var maxDelta = newDiff * maxPercent;
 
-	scale.options.ticks.min = rangeMinLimiter(zoomOptions, scale.min + minDelta);
-	scale.options.ticks.max = rangeMaxLimiter(zoomOptions, scale.max - maxDelta);
+	scale.options.min = rangeMinLimiter(zoomOptions, scale.min + minDelta);
+	scale.options.max = rangeMaxLimiter(zoomOptions, scale.max - maxDelta);
 }
 
 function zoomTimeScale(scale, zoom, center, zoomOptions) {
```

Comment:
you could remove this function now and call `zoomNumericalScale` where it's called since that's all it does anyway

Reply:
Same with `panTimeScale` below

Reply:
> 
> 
> you could remove this function now and call `zoomNumericalScale` where it's called since that's all it does anyway

done. I'm still having a look but I decided to push anyway the current updates

Reply:
> 
> 
> Same with `panTimeScale` below

I think you wanted to write `panCategoryScale`, doesn't it?

Reply:
What I mean is `panTimeScale` just calls `panNumericalScale` directly. You could just delete `panTimeScale` and call `panNumericalScale` where `panTimeScale` is used to avoid the extra method that no longer is really necessary

Reply:
done, removed also `zoomTimeScale`